i18n/en-US/music/queue.json

```diff
@@ -0,0 +1,8 @@
+{
+    "DESCRIPTION": "Displays a list of videos queued to stream.",
+    "USAGE": "queue",
+    "NONE": "**__Queue:__** There are no videos in the queue.",
+    "CURRENT": "**__Currently Streaming:__** **{{title}}** ({{time}) | Requested by **{{user}}**",
+    "LIST": "● **{{title}}** ({{time}) | Requested by **{{user}}**",
```

Comment:
After uploading the strings to Crowdin, these special characters will not render properly and/or become other characters. Can we use unicode instead? The unicode for `●` is `\u2022`.